### PR TITLE
Boardid 6

### PR DESCRIPTION
**Pull request by virajkaram**
Processing for new boardid=6, with a bad channel. The images are masked throughout processing, except in the pre-stack stage where swarp cannot handle the masked values distributed throughout the image, producing a fully masked image.
To get around this, the masked pixels are replaced with the median of the entire image in the pre-stack step. 
More sophisticated options (local median instead of global median?) will be explored in future PRs.
Also updates the way subdetid dataframe is calculated to ensure compatibility with existing boardid->subdetid mapping.

### CODE REVIEW

**Review by robertdstein — CHANGES_REQUESTED** [review]

File: mirar/pipelines/winter/load_winter_image.py [hunk 1]
```diff
@@ -425,12 +429,6 @@ def get_raw_winter_mask(image: Image) -> np.ndarray:
     header = image.header
 
     mask = np.zeros(data.shape)
-    if header["BOARD_ID"] == 0:
```
[the remark this answers is not in the document]
Comment (virajkaram) [reply mirar/pipelines/winter/load_winter_image.py:428]: done

File: mirar/pipelines/winter/constants.py [hunk 2]
```diff
@@ -34,7 +34,9 @@
 
 _subdets = []
 
-for ndetector in range(WINTER_N_BOARDS):
+winter_board_ids = [1, 2, 3, 4, 5, 6]
```
[the remark this answers is not in the document]
Comment (virajkaram) [reply mirar/pipelines/winter/constants.py]: now outdated, done

File: mirar/pipelines/winter/load_winter_image.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (virajkaram) [reply mirar/pipelines/winter/load_winter_image.py]: done

File: mirar/pipelines/winter/load_winter_image.py [hunk 3]
```diff
@@ -198,6 +199,9 @@ def clean_header(header: fits.Header) -> fits.Header:
 
     try:
         header["BOARD_ID"] = int(header["BOARD_ID"])
+        assert (
```
[the remark this answers is not in the document]
Comment (virajkaram) [reply mirar/pipelines/winter/load_winter_image.py:202]: done, now includes all boards

**Review by robertdstein — APPROVED** [review]

**Pull request merged**